lib/tasks/lumen.rake

```diff
@@ -737,6 +738,34 @@ where works.id in (
 
     puts "#{date_time_task.call} Finishing the task"
   end
+
+  desc 'Precomputation for word cloud'
+  task sampling_and_weight_calculation: :environment do
+    sample_count = 100
```

Comment:
Let's make it possible to read this from `ENV` while providing a default, something like `ENV.fetch('WORD_CLOUD_SAMPLE_SIZE', 100)` -- that will make it easy for us to test different options.